schema.graphql

```diff
@@ -593,9 +593,18 @@ type Transfer @entity(immutable: true) {
   blockTimestamp: BigInt!
 }
 
+type License @entity {
+  "Unique identifier made up of licenseType"
+  id: ID!
+  "Timestamp of last update"
+  updatedAt: BigInt!
+}
+
 type Dependency @entity {
   "Unique identifier made up of dependency name and version separated by an @ symbol (e.g. p5js@1.0.0)"
   id: ID!
+  "License Type of this dependency"
+  licenseType: String!
```

Comment:
@yoshiwarab this could potentially be type `License!`, right?

Reply:
yup!